gruntfile.js

```diff
@@ -537,4 +439,11 @@ module.exports = function (grunt) {
       }
     }
   });
+
+  grunt.registerTask('vendor', 'vendor:<min|reg>', function(min) {
```

Comment:
That's enough, just note that following array:
```yaml
js:
   - path/to/vendor.js
```
Will be available in `src.jsVendor` property.